### PR TITLE
programatically fix straight quotes/apostrophes

### PR DESCRIPTION
**Pull request by jaredh159**

### CODE REVIEW

File: journal/modernized/08-chapter-7.adoc [hunk 1]
```diff
@@ -785,7 +785,7 @@ had a religious opportunity in this family,
 which closed my services for the present in these parts.
 
 12th.--Rode forty-seven miles to Cadiz, where we lodged.
-Next day, about ten, got to David Binns', at Harrisville.
+Next day, about ten, got to David Binns'`, at Harrisville.
```
Comment (Henderjay) [inline journal/modernized/08-chapter-7.adoc]: Because of the comma, this is going the wrong way.

File: journal/modernized/09-chapter-8.adoc [hunk 2]
```diff
@@ -1093,7 +1093,7 @@ I felt greatly relieved after this meeting and very peaceful in mind.
 Many Friends spoke affectionately to us after meeting,
 toward whom the salutation of gospel love was felt to flow without respect of persons.
 
-Dined at Henry Roberts', where many Friends came.
+Dined at Henry Roberts'`, where many Friends came.
```
Comment (Henderjay) [inline journal/modernized/09-chapter-8.adoc]: Same here

File: journal/modernized/09-chapter-8.adoc [hunk 3]
```diff
@@ -1121,7 +1121,7 @@ A woman Friend from a neighboring Quarterly Meeting,
 took up most of the time in preaching and praying.
 It was a very trying meeting to me, believing the cause was hurt by this individual.
 
-Dined at Josiah Evans'. After a religious opportunity in this family,
+Dined at Josiah Evans'`. After a religious opportunity in this family,
```
Comment (Henderjay) [inline journal/modernized/09-chapter-8.adoc]: Same here

File: journal/modernized/09-chapter-8.adoc [hunk 4]
```diff
@@ -1188,7 +1188,7 @@ but had nothing to communicate.
 Lodged at Joseph Walton's. Believing it to be required of me
 to visit the meetings in the limits of Salem Quarter,
 we set out on Second-day afternoon the 26th, for Woodbury;
-got there in the evening and lodged at Carlton Stokes'. That night I was quite ill,
+got there in the evening and lodged at Carlton Stokes'`. That night I was quite ill,
```
Comment (Henderjay) [inline journal/modernized/09-chapter-8.adoc]: and here

File: journal/modernized/10-chapter-9.adoc [hunk 5]
```diff
@@ -962,12 +962,12 @@ which I and my companion, Abigail Sears, concluded to do. Before leaving,
...
 if he thought it was contrary to Scripture for a woman to preach.
 He replied, "`No; I know some people do, but I have considered this passage of Scripture:
-'Male and female all one in Christ;' and I do not consider it
+'`Male and female all one in Christ;`' and I do not consider it
 contrary to Scripture for a woman to preach.`"
 He was a sober, serious countenanced man,
 and I had some interesting conversation with him and his wife before leaving.
 Where Abigail and I lodged, we found them very kind, and glad of our company.
-Next morning we started on our way back to Coal Creek, and got to Job Briggs',
+Next morning we started on our way back to Coal Creek, and got to Job Briggs'`,
```
Comment (Henderjay) [inline journal/modernized/10-chapter-9.adoc]: and here

File: journal/modernized/10-chapter-9.adoc [hunk 6]
```diff
@@ -1008,7 +1008,7 @@ The first day we rode about forty miles, and got to the house of Thomas and Mary
 in Benton County, where a few families of Friends are settled.
 
 On the next evening we had an appointed meeting,
-held at a school house not far from Thomas Emmons'. It being harvest time,
+held at a school house not far from Thomas Emmons'`. It being harvest time,
```
Comment (Henderjay) [inline journal/modernized/10-chapter-9.adoc]: here too

File: journal/original/08-chapter-7.adoc [hunk 7]
```diff
@@ -786,7 +786,7 @@ had a religious opportunity in this family,
 which closed my services for the present in these parts.
 
 12th.--Rode forty-seven miles to Cadiz, where we lodged.
-Next day, about ten, got to David Binns', at Harrisville.
+Next day, about ten, got to David Binns'`, at Harrisville.
```
Comment (Henderjay) [inline journal/original/08-chapter-7.adoc]: and here

File: journal/original/09-chapter-8.adoc [hunk 8]
```diff
@@ -1091,7 +1091,7 @@ I felt greatly relieved after this meeting and very peaceful in mind.
 Many Friends spoke affectionately to us after meeting,
 toward whom the salutation of gospel love was felt to flow without respect of persons.
 
-Dined at Henry Roberts', where many Friends came.
+Dined at Henry Roberts'`, where many Friends came.
```
Comment (Henderjay) [inline journal/original/09-chapter-8.adoc]: and here

File: journal/original/09-chapter-8.adoc [hunk 9]
```diff
@@ -1119,7 +1119,7 @@ A woman Friend from a neighboring Quarterly Meeting,
 took up most of the time in preaching and praying.
 It was a very trying meeting to me, believing the cause was hurt by this individual.
 
-Dined at Josiah Evans'. After a religious opportunity in this family,
+Dined at Josiah Evans'`. After a religious opportunity in this family,
```
Comment (Henderjay) [inline journal/original/09-chapter-8.adoc]: here too

File: journal/original/09-chapter-8.adoc [hunk 10]
```diff
@@ -1186,7 +1186,7 @@ but had nothing to communicate.
 Lodged at Joseph Walton's. Believing it to be required of me
 to visit the meetings in the limits of Salem Quarter,
 we set out on Second-day afternoon the 26th, for Woodbury;
-got there in the evening and lodged at Carlton Stokes'. That night I was quite ill,
+got there in the evening and lodged at Carlton Stokes'`. That night I was quite ill,
```
Comment (Henderjay) [inline journal/original/09-chapter-8.adoc]: here too

File: journal/original/10-chapter-9.adoc [hunk 11]
```diff
@@ -961,12 +961,12 @@ which I and my companion, Abigail Sears, concluded to do. Before leaving,
...
 if he thought it was contrary to Scripture for a woman to preach.
 He replied, "`No; I know some people do, but I have considered this passage of Scripture:
-'Male and female all one in Christ;' and I do not consider it
+'`Male and female all one in Christ;`' and I do not consider it
 contrary to Scripture for a woman to preach.`"
 He was a sober, serious countenanced man,
 and I had some interesting conversation with him and his wife before leaving.
 Where Abigail and I lodged, we found them very kind, and glad of our company.
-Next morning we started on our way back to Coal Creek, and got to Job Briggs',
+Next morning we started on our way back to Coal Creek, and got to Job Briggs'`,
```
Comment (Henderjay) [inline journal/original/10-chapter-9.adoc]: and here

File: journal/original/10-chapter-9.adoc [hunk 12]
```diff
@@ -1007,7 +1007,7 @@ The first day we rode about forty miles, and got to the house of Thomas and Mary
 in Benton County, where a few families of Friends are settled.
 
 On the next evening we had an appointed meeting,
-held at a school house not far from Thomas Emmons'. It being harvest time,
+held at a school house not far from Thomas Emmons'`. It being harvest time,
```
Comment (Henderjay) [inline journal/original/10-chapter-9.adoc]: yup, here too

**Review by Henderjay — COMMENTED** [review]
I think there is only just the one error... but in multiple places.

**Review by Henderjay — COMMENTED** [review]
Don't see any problems with these. I didn't look at them all, but a large number of them.

**Comment by jaredh159** [comment]
I fixed the ones you pointed out, so I think I'm just going to merge this, because we've both looked pretty carefully at it. Thanks.

**Pull request merged**